paimon-core/src/main/java/org/apache/paimon/catalog/AbstractCatalog.java

```diff
@@ -173,6 +173,12 @@ public List<String> listTables(String databaseName) throws DatabaseNotExistExcep
     @Override
     public void dropTable(Identifier identifier, boolean ignoreIfNotExists)
             throws TableNotExistException {
+        dropTable(identifier, ignoreIfNotExists, false);
```

Comment:
ifPurge should be true.
Here should have some test to verify it.

Reply:
already add some tests for it

Reply:
already add some tests for it